deployment_scripts/appimage/build_appimage.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+python_cmd="python3.7"
+APPDIR="$HOME/monkey-appdir"
+INSTALL_DIR="$APPDIR/usr/src"
```

Comment:
Is there a specific reason we want `/usr/src`?

Reply:
It's an AppImage convention.